opentelemetry-sdk/benches/log_enabled.rs

```diff
@@ -5,10 +5,12 @@
     Total Number of Cores:   14 (10 performance and 4 efficiency)
     | Test                                         | Average time|
     |---------------------------------------------|-------------|
-    | exporter_disabled_concurrent_processor      |  1.9 ns     |
-    | exporter_disabled_simple_processor          |  5.0 ns     |
+    | exporter_disabled_concurrent_processor      |  1.0 ns     |
```

Comment:
Add the inline attribute to the processor and exporter's `event_enabled` methods as well?

Reply:
I tried that but it didn't give any boost 🤷‍♂️